components/object-table/src/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'govuk-frederic';
+
+import { objectHasValueForKeys, keysFromFields, rowsFromObject } from '@govuk-frederic/utils';
+
+
+/**
+ *
+ * ### Usage
+ * 
+ * Simple
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ *   { key: 'three', heading: 'Three' },
+ *   { key: 'four', heading: 'Four' },
+ * ];
+ * const object = { one: 'test', two: 'test', three: '', four: null };
+ * const title = ['Heading'];
+ * 
+ * <ObjectTable fields={fields} object={object} title={title}/>;
+ * ```
+ * 
+ * With skipEmptyValues
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ *   { key: 'three', heading: 'Three' },
+ *   { key: 'four', heading: 'Four' },
+ * ];
+ * const object = { one: 'test', two: 'test', three: '', four: null };
+ * const title = ['Heading'];
+ * 
+ * <ObjectTable fields={fields} object={object} title={title} skipEmptyValues={false}/>
+ * ```
+ * 
+ * With hideWithNoValues
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ * ];
+ * const object = { };
+ * const title = ['Heading'];
+ *
+ * <ObjectTable fields={fields} object={object} title={title} hideWithNoValues />;
+ * ```
+ */
+const ObjectTable = ({
+  title,
+  fields = [],
+  object = {},
+  hideWithNoValues = false,
```

Comment:
as per `ArrayObjectTable` we should consider using `defaultProps` for these two flags

Reply:
@stevesims Good point - probably better to do things the "React way"! I've added a commit for this too and pushed.